test/ClaimerAgent.t.sol

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.17;
+
+import "forge-std/console2.sol";
+
+import "forge-std/Test.sol";
+
+import { Environment, PrizePool, Vault, GasConfig, Claimer } from "src/Environment.sol";
+import { ClaimerAgent } from "src/ClaimerAgent.sol";
+
+contract ClaimerAgentTest is Test {
+
+    Environment env = Environment(address(0x1));
+    PrizePool prizePool = PrizePool(address(0x2));
+    Vault vault = Vault(address(0x5));
+    Claimer claimer = Claimer(address(0x6));
+
+    address user1 = address(0x3);
+    address user2 = address(0x4);
+    
+    ClaimerAgent agent;
+
+    uint numTiers = 2;
+
+    function setUp() public {
+        vm.etch(address(env), "environment");
+        vm.etch(address(prizePool), "prizePool");
+        vm.etch(address(vault), "vault");
+        vm.etch(address(claimer), "claimer");
```

Comment:
Getting an error when running tests saying:
`[FAIL. Reason: Setup failed: Etch cannot be used on precompile addresses (N < 10). Please use an address bigger than 10 instead]`

Reply:
forge version: `forge 0.2.0 (6defdce 2023-05-17T00:04:12.304272526Z)`